### ISSUE TITLE
About the accuracy

### ISSUE THREAD

**Issue by 8Grace8** [ronghangzhu/CensNet#1]
I download your code, and run it with setting train_ratio=0.03 for 10 times, but the mean accuracy I've got is lower a lot than the accuracy you offered in your paper.Is there any parameters are supposed to be changed or is there anything to be paid attention to? Thank you so much!

**Comment by Fatigerrr** [comment]
I think there is sth wrong  in his code.it seems that "M1 = mask1 * torch.ones(multiplier1.shape[1]).cuda() + (1. - mask1) * multiplier1" is incorrect
